Source/Fuse.Launcher.InterApp/InterApp/InterAppLauncher.uno

```diff
@@ -40,7 +40,7 @@ namespace Fuse.LauncherImpl
 			}
 		}
 
-		public static void LaunchApp(string uri)
+		public static void LaunchApp(string uri, string appStoreUri)
```

Comment:
Can the `appStoreUri` parameter be optional, as in `string appStoreUri = null`?

Reply:
Sure can, wasn't sure we could do that in Uno, when's the Uno Handbook coming out? 😄

Reply:
Haha :D